tests/src/python/test_qgslayoutlegend.py

```diff
@@ -380,7 +380,11 @@ def testSymbolExpressions(self):
         layer = QgsProject.instance().addMapLayer(point_layer)
         legendlayer = legend.model().rootGroup().addLayer(point_layer)
 
-        point_layer.countSymbolFeatures(disable_async=True)
+        loop = QEventLoop()
+        counterTask = point_layer.countSymbolFeatures()
+        counterTask.symbolsCounted.connect(loop.quit)
```

Comment:
I would probably switch lines 384 and 385 to avoid the risk of having the task completed before the connection is established.
And feel free to remove the line below which was meant to be substituted by `counterTask = point_layer.countSymbolFeatures()` :)

Reply:
done ty